modules/theme/src/themes/default/assets/images/icons/lock-recover-icon.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="14" height="14" viewBox="0 0 14 14">

```

Comment:
Add the license header and indent with `4` spaces.
```suggestion
<svg class="icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="14" height="14" viewBox="0 0 14 14">

```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/2242/commits/75dc29ea9f1e707315a58da2d039ae0a61c32c38